doc/uml/deployment.puml

```diff
@@ -0,0 +1,25 @@
+@startuml DeploymentDiagram
+
+node "Polarion"<<virtual-machine>> as polarion
+database "SVN Repository" as svnRepo
+
+interface "SOAP API" as polarionSoapApi
+
+polarion --- polarionSoapApi
+polarion -- svnRepo
+
+node "Continuous Integration Server"<<virtual-machine>> {
+
+    package "scripts" {
+        component "pyPolarionCli" as pyPolarionCli
+    }
+}
+
+polarionSoapApi )-- pyPolarionCli: <<uses>>

```

Comment:
Remove stereotype, not required for client/server interface.

Reply:
![image](https://github.com/user-attachments/assets/0a72a5fa-f79e-40c9-bf8c-a2aaac60e55e)
It was present in the deployment of pyMetricCli. Should I remove it from everywhere?

Reply:
yep